index.html

```diff
@@ -0,0 +1,380 @@
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+			<div class="details-card details2">
+				<div class="project-image4"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Multi-Post Stories</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="Cinf">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="pl">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+		</section>
+
+		<!-- About Me section -->
+
+		<section id="section-about">
+			<div class="about">
+				<h1 class="about-title">About Myself</h1>
+				<div class="self-description self-desc">
+					<p>
+						I’m a software developer! I can help you build a product , feature or website Look through some
+						of
+						my work and
+						experience! If you like what you see and have a project you need coded, don’t hestiate to
+						contact
+						me.
+					</p>
+				</div>
+				<a class="btn-mediaConnect connectm" href="#">LET’S CONNECT</a>
+				<ul class="media-list listm">
+					<li>
+						<a href="" class="media-link"><img src="./images/twitterIcon.svg" alt="twitterIcon"
+								class="media-icon"></a>
+					</li>
+					<li>
+						<a href="" class="media-link"><img src="./images/linkedIncon.svg" alt="linkdlnIcon"
+								class="media-icon"></a>
+					</li>
+					<li>
+						<a href="" class="media-link"><img src="./images/mailIcon.svg" alt="mailIcon"
+								class="media-icon"></a>
+					</li>
+					<li>
+						<a href="" class="media-link"><img src="./images/githubIcon.svg" alt="githubIcon"
+								class="media-icon"></a>
+					</li>
+					<li>
+						<a href="" class="media-link"><img src="./images/inconito.svg" alt="inconitoIcon"
+								class="media-icon"></a>
+					</li>
+				</ul>
+				<div class="btn-resume">
+					<a href="">Get my resume</a>
+				</div>
+			</div>
+			<div class="skill-list">
+				<ul class="sl">
+
+					<!-- Languages -->
+
+					<li>
+						<details class="list-container">
+							<summary class="list-title">Languages</summary>
+							<ul class="list-item">
+								<li>
+									<img class="list-icon" src="./images/jsIcon.svg" alt="iconjs">
+								</li>
+								<li>
+									javascript
+								</li>
+							</ul>
+							<ul class="list-item">
+								<li>
+									<img class="list-icon" src="./images/htmlicon.svg" alt="iconhtml">
+								</li>
+								<li>
+									Html
+								</li>
+							</ul>
```

Comment:
- [ ] I noticed that your design here is missing the `tech stack icons`. Please include them in your design. The links might be broken since they are not showing on your desktop design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/220821798-50d74c1b-3bb5-4280-8712-575cbe4d228c.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/220821856-9062ca38-b759-4c87-97a8-2275ae16af45.png)